client-src/elements/chromedash-drawer.js

```diff
@@ -0,0 +1,165 @@
+          border-bottom: var(--card-border);
+          box-shadow: var(--card-box-shadow);
+          align-items: center;
+          margin: 0 var(--content-padding);
+          -webkit-font-smoothing: initial;
+        }
+        nav a {
+          text-decoration: none !important;
+          cursor: pointer;
+          font-size: var(--nav-link-font-size);
+          text-align: left;
+          padding: var(--content-padding-half) var(--content-padding);
+          color: var(--nav-link-color);
+          white-space: nowrap;
+          border-bottom: var(--nav-link-border);
+        }
+        nav a:hover {
+          color: black;
+          background: var(--nav-link-hover-background);
+        }
+        nav a.disabled {
+          opacity: 0.5;
+          pointer-events: none;
+        }
+        nav [active] {
+          color: var(--nav-link-active-color);
+          border-bottom: var(--nav-link-active-border);
+        }
+        nav [active] a {
+          color: var(--nav-link-active-color);
+        }
+        nav .nav-dropdown-container {
+          position: relative;
+        }
+        nav .nav-dropdown-container ul {
+          display: none;
+          top: 80%;
+          left: 10px;
+          z-index: 1;
+          text-align: left;
+          background: var(--card-background);
+          border-bottom: var(--card-border);
+          box-shadow: var(--card-box-shadow);
+        }
+        nav .nav-dropdown-container ul {
+          padding-left: 10px;
+        }
+        nav .nav-dropdown-container a {
+          display: block;
+        }
+        nav .nav-dropdown-container .nav-dropdown-trigger:hover + ul,
+        nav .nav-dropdown-container ul:hover {
+          display: block;
+        }
+        .flex-item {
+          padding-top: 10px;
+          padding-bottom: 10px;
+          text-align: left;
+        }
+        .flex-item-inner {
+          margin-top: -10px;
+        }
+        sl-drawer a {
+          display: block;
+        }
+        @media only screen and (max-width: 700px) {
+          header {
+            --logoSize: 24px;
+
+            margin: 0;
+            display: block;
+          }
+          sl-drawer a {
+            display: block;
+          }
+        }
+    `];
+  }
+
+  static get properties() {
+    return {
+      currentPage: {type: String},
+      user: {type: Object},
+      loading: {type: Boolean},
+    };
+  }
+
+  constructor() {
+    super();
+    this.currentPage = '';
+    this.user = {};
+    this.loading = false;
+  }
+
+  isCurrentPage(href) {
+    return this.currentPage.startsWith(href);
+  }
+
+  handleDrawerActions() {
+    const drawer = this.shadowRoot.querySelector('.drawer-placement-start');
+    if (drawer.open) {
+      drawer.hide();
+    } else {
+      drawer.show();
+    }
+  }
+
+  renderTabs() {
+    return html`
+      <sl-drawer label="Menu" placement="start" open=true class="drawer-placement-start" contained>
```

Comment:
Can we hide the "X" icon and the whole row that says "Menu" by specifying attribute "no header"?  The user can already toggle the menu closed by clicking the hamburger icon again.

Reply:
done